frb_example/benches/dart/web/uuids.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```

Comment:
Based on your feedback I checked whether to use a conditional `<script>` tag in HTML and yes indeed it seems possible.
I'm gonna do this a little bit later on (currently at the airport so my timeframe is limited). I think I'll have to use some env var to conditionally dictate which `.js` file to load on startup, AFAIU.

Reply:
take your time!

Reply:
As a reminder, there's actually no way to rely on env var since it's a potential security threat (based on various feedbacks from StackOverflow). Currently looking into doing the same trick with query string parameter, but couldn't get it to run successfully this morning. Still pending.